gc/base/EnvironmentBase.hpp

```diff
@@ -562,6 +589,49 @@ class MM_EnvironmentBase : public MM_BaseVirtual
 	MMINLINE void setAllocationColor(uint32_t allocationColor) { _allocationColor = allocationColor; }
 
 	MMINLINE MM_WorkStack *getWorkStack() { return &_workStack; }
+
+	/**
+	 * Get a pointer to common GC metadata attached to this environment. The GC environment structure
+	 * is defined by the client language and bound to the environment delegate attached to this class.
+	 * It is typically used to hold thread-local GC-related stats and caches that are merged at key
+	 * points into global structures.
+	 */
+	MMINLINE GC_Environment *getGCEnvironment() { return _delegate.getGCEnvironment(); }
+
+	/**
+	 * Called on each participating GC thread before starting mark phase. This can be used
+	 * to reset local thread stats and reference buffers prior to commencing marking.
+	 */
+	MMINLINE void
+	markingPhaseStart()
```

Comment:
Methods renamed.

MM_EnvironmentBase:
  markingPhaseStart() -> markingPhaseStarted()
  markingPhaseFinish() -> markingPhaseFinished()
  
MM_EnvironmentDelegate: (omr/glue)
  markStart() -> markingStarted()
  markFinish() -> markingFinished

Similarly for other language environment delegates.